src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -373,6 +477,18 @@ public void Patch(TStructuralType original)
             CopyChangedValues(original);
         }
 
+        /// <summary>
+        /// Overwrites the <paramref name="original"/> entity with the changes tracked by this Delta.
+        /// <remarks>The semantics of this operation are equivalent to a HTTP PATCH operation, hence the name.</remarks>
+        /// </summary>
+        /// <param name="original">The entity to be updated.</param>
+        /// <param name="patchHandler">Patch Handler</param>
+        public void Patch(TStructuralType original, IPatchMethodHandler patchHandler)
+        {
+            PatchHandler = patchHandler as PatchMethodHandler<TStructuralType>;
```

Comment:
>PatchHandler [](start = 12, length = 12)

Rather than have PatchHandler as an internal instance property, would it be better to have an overload of CopyChangedValues that took the PatchHandler as an input parameter?